docs/core/entity/alarm-control-panel.md

```diff
@@ -86,6 +87,21 @@ class MyAlarm(AlarmControlPanelEntity):
         """Send arm away command."""
 ```
 
+### Alarm Arm Vacation
+
+Send arm vacation command.
+
+```python
+class MyAlarm(AlarmControlPanelEntity):
+    # Implement one of these methods.
+
+    def alarm_arm_vacation(self, code=None) -> None:
+        """Send arm vacation command."""
+
+    def async_alarm_arm_vacation(self, code=None) -> None:
```

Comment:
```suggestion
    async def async_alarm_arm_vacation(self, code=None) -> None:
```

Reply:
Fine by me, but i just copied other alarm state code so these should be altered aswell?

Reply:
Yes, it would be good to fix that. Thanks!